integration/krew/index.go

```diff
@@ -0,0 +1,85 @@
+package krew
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+
+	"github.com/golang/glog"
+	"sigs.k8s.io/krew/cmd/krew/cmd"
+)
+
+const (
+	persistentIndexCache = "krew-persistent-index-cache"
+)
+
+var (
+	once     sync.Once
+	indexTar []byte
+)
+
+// InitializeIndex initializes the krew index in `$root/index` with the actual krew-index.
+// It caches the index tree as in-memory tar after the first run.
```

Comment:
Hmm why don’t we do a “krew update” every time instead?
Are you worried about each test cloning the repo over and over again?

Reply:
Yes, repeated cloning will make the tests awfully slow. Especially when iterating on tests, this is painful.